test/e2e/origins.test.js

```diff
@@ -1,45 +1,46 @@
 import provider from 'eth-provider'
+import { hexToNumber, numberToHex } from 'web3-utils'
 
 jest.mock('../../main/store/persist')
 
 let frame
 
 beforeEach(done => {
-  frame = provider('frame', { origin: 'frame.test' })
+  frame = provider('frame')
   frame.once('connect', () => {
     frame.request({ method: 'eth_accounts', params: [] }).then(() => done())
   })
-}, 10 * 1000)
+})
 
 afterEach(() => {
   frame.removeAllListeners('chainChanged')
   frame.close()
 })
 
 it('should be able to change the chain for a given origin', async () => {
-  const [chains, currentChain] = await Promise.all([
-    frame.request({ method: 'wallet_getChains' }),
+  const [chains, currentChainId] = await Promise.all([
+    frame.request({ method: 'wallet_getEthereumChains' }),
     frame.request({ method: 'eth_chainId' })
   ])
 
-  const targetChain = chains.find(c => c !== currentChain)
+  const targetChain = chains.find(c => c.chainId !== hexToNumber(currentChainId))
 
   if (!targetChain) throw new Error('no available chains to switch to!')
 
   return new Promise((resolve, reject) => {
     frame.on('chainChanged', async updatedChainId => {
       try {
-        expect(updatedChainId).toBe(targetChain)
+        expect(updatedChainId).toBe(numberToHex(targetChain.chainId))
 
         const chainId = await frame.request({ method: 'eth_chainId' })
-        expect(chainId).toBe(targetChain)
+        expect(chainId).toBe(numberToHex(targetChain.chainId))
         resolve()
       } catch (e) { reject(e) }
     })
     
     frame.request({
       method: 'wallet_switchEthereumChain',
-      params: [{ chainId: targetChain }]
+      params: [{ chainId: targetChain.chainId }]
     })
   })
 }, 5 * 1000)
```

Comment:
should we remove this custom timeout override?

Reply:
Why?

Reply:
because we added the longer timeout to all the e2e tests, this is probably not necessary now

Reply:
You added it in #870

Reply:
So I guess it's up to whether we believe removing this shorter timeout is an improvement

Reply:
Just trying to understand why this one had a shorter timeout in the first place - all the others were 30s and just because I'm moving those 30s to be the default doesn't change the reasoning for this one being less than that.